### DISCUSSION TITLE
Any examples to add an item in side '+' menu?

### DISCUSSION THREAD

**Discussion by kuldeepk** [outline/rich-markdown-editor#214] [General]
Hi,

Great editor! Thanks for open sourcing it.

I am trying to add a menu item in the menu that shows up by click a '+' sign on left. Does that go in extensions? I understood how to add 'embeds' in the menu, but what I'm trying to add is something custom - a fancy table item. Are there a good examples on how to add that?

Thanks in advance!

**Comment by tommoor** [comment]
This isn't possible right now, I'd recommend you fork.

**Reply by kuldeepk** [reply]
Gotcha. Thanks!
